irohad/main/impl/raw_block_loader.cpp

```diff
@@ -35,7 +35,7 @@ namespace iroha {
         const std::string &data) {
       return jsonToProto<iroha::protocol::Block>(data) | [](auto &&block) {
         return boost::optional<std::shared_ptr<Block>>(
-            std::make_shared<shared_model::proto::Block>(std::move(block)));
+            std::make_shared<shared_model::proto::Block>(std::move(block.block_v1())));
```

Comment:
Should we prepare the method somehow for parsing future block versions?

Probably we need to place some stub with logging for other and unsupported versions.